debezium-quarkus-outbox/runtime/pom.xml

```diff
@@ -27,6 +27,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-opentracing</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
```

Comment:
Why is this one needed?

Reply:
When I was building against 2.0.0.Alpha3, I was getting a failure that a runtime dependency was missing for the resteasy deployment artifact.  Adding this dependency resolved that failure and allowed it to build.

Reply:
Hum, that's a bit weird, no? This extension shouldn't have any dependency on RESTEasy, or should it?

Reply:
That's a great question and it seems we don't need it nor the deployment artifact that triggered this dependency in the first place.  That deployment artifact was there since the beginning, so perhaps either a mistake from the get-go or something changed with Quarkus that we never caught.  Either way, I'll remove RESTEasy from both the runtime and deployment artifacts.